keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,6 +3,8 @@
 ## Table of Contents
 
 <!-- toc -->
+- [Object Storage Support](#object-storage-support)
```

Comment:
I don't understand the point of these 2 links. Is this a kep template requirement?

Reply:
The TOC contents were autogenerated.  If it's incorrect, it's likely due to a formatting issue in the contents of the doc itself.  I found one of these when I was doing the edits because the TOC update was so outrageous.  I'll see if I can track this one down.

Reply:
I don't know what changed between this pr and the merged kep, but perhaps it's the "Object Storage Support" section, which is currently empty and I don't understand its purpose.

Reply:
What's changed is the TOC was updated by a tool in the PR.  Every section header generally goes into the TOC.